src/lib/SmartTokens/Types/Constants.hs

```diff
@@ -11,23 +12,24 @@ module SmartTokens.Types.Constants(
 ) where
 
 import Plutarch.LedgerApi.V1 (PTokenName (..))
-import Plutarch.Prelude (ClosedTerm, PAsData, pconstant, pconstantData)
+import Plutarch.Prelude (ClosedTerm, PAsData, pconstant)
 import PlutusLedgerApi.V1 (TokenName (..))
+import PlutusTx.Builtins.HasOpaque (stringToBuiltinByteString)
 
 protocolParamsToken :: TokenName
-protocolParamsToken = "ProtocolParams"
+protocolParamsToken = TokenName (stringToBuiltinByteString "ProtocolParams")
```

Comment:
Why do we need `stringToBuiltinByteString` here, but not in `directoryNodeToken`?

Reply:
@colll78